routes.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const app = express();
+const port = 5000;
+
+const data = [
+  {
+    "NAME": "Janhavi",
+    "ROLL NO.": 11111,
+    "PHONE NO.": "911122334",
+    "MEME": "Meme",
+    "PHOTOGRAPH": "Photo",
+    "REASON": "Reason",
+    "HOUR SLEPT": 2
+  },
+  
+];
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
```

Comment:
Consider adding error handling for your routes. This will allow you to handle any potential issues that may arise and send a proper response to the client. [medium]